config/initializers/social_share_button.rb

```diff
@@ -4,5 +4,5 @@
 # found here: https://github.com/huacnlee/social-share-button#configure
 #
 SocialShareButton.configure do |config|
-  config.allow_sites = %w(twitter facebook google_plus)
+  config.allow_sites = %w(twitter facebook)
```

Comment:
we should add all social networks, whatsapp, telegram, telegram web, ...
again. Let's copy all configuration from the Upgrade notes please.